README.md

```diff
@@ -54,9 +79,82 @@ about "Cannot resolve module 'source-map-loader'" due to shortcomings in webpack
 Deployment
 ==========
 
+Configure the app by modifying the `config.json` file to the correct values:
+
+1. default_hs_url is for the home server url (could be http://your.server.ip:8008 if vector and synapse are on the same machine),
+2. default_is_url is for the identity server (could be empty but if you let it empty, email notifications and email login may not work).
+
+
 Just run `npm run build` and then mount the `vector` directory on your webserver to
 actually serve up the app, which is entirely static content.
 
+The port 8008 is used for http and 8448 for https. Set `http://your.server.ip:8008` or `https://your.server.ip:8448`
+
+If you want to expose your service outside, don't forget to open the port in your firewall / box
+
+Example of apache vhost configuration:
+
+```
+<VirtualHost *:80>
+	ServerAdmin webmaster@localhost
+
+	DocumentRoot /var/www/vector-web/vector
+	<Directory />
+		Options FollowSymLinks
+		AllowOverride None
+	</Directory>
+	<Directory /var/www/vector-web/vector/>
+		Options Indexes FollowSymLinks MultiViews
+		AllowOverride None
+		Order allow,deny
+		allow from all
+	</Directory>
+
+	ErrorLog ${APACHE_LOG_DIR}/error.log
+
+	# Possible values include: debug, info, notice, warn, error, crit,
+	# alert, emerg.
+	LogLevel warn
+
+	CustomLog ${APACHE_LOG_DIR}/access.log combined
+</VirtualHost>
+```
+
+You can add a reverse proxy if you do not want to expose your server on a new port.
+
+```
+$ sudo a2enmod proxy
+$ sudo a2enmod proxy_http
+```
+
+And add this to inside your apache vhost:
+
+```
+ProxyRequests Off
+ProxyPreservehost on
+ServerName  your.server.name
+ServerAlias www.your.server.name                # optional
+ProxyPass /matrix http://127.0.0.1:8008         # do not add a slash after the port number
+ProxyPassReverse /matrix http://127.0.0.1:8008  # do not add a slash after the port number
```

Comment:
Hmm - this is really very specific to Home Server setup and I'm not really convinced it belongs in Vector's README? Perhaps this could go in synapse's README, or maybe a separate document that goes through setting up a synapse server and a vector behind an Apache in detail would be more useful?